docs/content/stories/DC.rst

```diff
@@ -5,24 +5,28 @@ DC resistivity
 
 .. topic:: Purpose
 
-	Understand basic setup and physics of a direct current (DC) resistivity survey within the context of a kimberlite exploration. Run DC forward modelling and inversion using SimPEG-Static package.
```

Comment:
fail is because the tag `.. _stories` in line 1 has been used elsewhere. You could use `TKC_DC_story` for example. 
![image](https://cloud.githubusercontent.com/assets/6361812/17311326/4b83f8da-57ff-11e6-833d-79f714c51184.png)
https://travis-ci.org/simpeg/tutorials/builds/148443880